### ISSUE TITLE
Community features

### ISSUE THREAD

**Issue by brettalcox** [brettalcox/logU-swift#23]
Planning for community features around 1.4+ release. Some thoughts: 
- Community dashboard - _In Progress_
- Community leaderboard - _In Progress_
- <strike>Community stats i.e. total sets, reps, poundage, etc.</strike> - Done
- _Potentially_ adding the ability to have "friends" or subscriptions to other lifters
- <strike>Updated privacy policy</strike> - Done
- <strike>Adding privacy options for not wanting to partake (GPS/gym logging)</strike> - Done
- Adding privacy options for not wanting to partake (leaderboards, friends/social, etc.)

**Comment by jaredbaj** [comment]
- Weigh ins with ability to add progress photo
- Form videos for top sets
- Link to IG for weightlifting videos
- Workout/routine template building
- Download popular routine packs/templates
- Goal tracking
- Funny ass messages
- Muscle targeting by lift
- Lifts grouped under bins like Deadlift > choose from Conventional Deadlift, Sumo Deadlift, Romanian Deadlift, etc... maybe just have base lifts and add modifiers like that idk

**Comment by brettalcox** [comment]
- <strike>Possibly, as an opt-in feature, can log gps and show on community world map.</strike> - Done
- <strike>Consolidating "Graphs" tab into a table row within the "User Stats" tab. This would save space, and allow for a graph to be created for every lift the user has done. The "Community" tab would replace the now-defunct "Graphs" tab. </strike> - _Done_

Community map has been finished. Map type has been changed from 'Standard' to 'Hybrid', which drastically reduces the amount of memory consumed/retained by map tiles and caching. Cluster image has been changed from green to (red: 0/255.0, green: 152/255.0, blue: 255/255.0, alpha: 1.0) to match the rest of the app.

EDIT - 3/22/2016: The GPS logging function has been overhauled. Instead of it grabbing the user's current location when the lift is logged (meaning a user could have lifts showing at the grocery store, gym, home, work, etc.-- cluttered, uninformative map), I have the user select their gym location in "Settings", and the lifts only log at that location. I used the LocationRow from the Eureka framework to easily implement setting the gym location. And then added the appropriate if/else/checking measures for the options enabled/disabled.

EDIT - 3/24/2016: A couple more features to be added in v1.4
- Average frequency per lifter
- Potentially a page showing the # of workouts per day/week total
- <strike>Weekly poundage and graph for the entire community</strike> - Done
- <strike>Potentially a target muscle graph</strike> - Done
- Average wilks score (maybe not considering some people have only logged like a 40kg squat)
